enable-vcp-image/enable-vcp-scripts/vcp-daemontset.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
+metadata:
+  name: vcp-status.vmware.com
+description: "POD status resource"
+versions:
+- name: v1
+---
+apiVersion: extensions/v1beta1
+kind: ThirdPartyResource
+metadata:
+  name: vcp-summary.vmware.com
+description: "Overall Summary of vSphere Cloud Provider configuration operations"
+versions:
+- name: v1
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: vcp-daementset
+  labels:
+    name: vcp-daementset
+  namespace: vmware
+spec:
+  template:
+    metadata:
+      name: vcp-pod
+      labels:
+        name: vcp-pod
+    spec:
+      serviceAccountName: vcpsa
+      containers:
+      - name: vcp-container
+        image: divyen/enablevcp:v1
```

Comment:
this for time being, once we have image published on the cna-storage, will replace with that cna-storage